ydb/src/grpc_wrapper/raw_topic_service/common/consumer.rs

```diff
@@ -0,0 +1,26 @@
+use std::collections::HashMap;
+
+use crate::grpc_wrapper::raw_topic_service::common::codecs::RawSupportedCodecs;
+use ydb_grpc::google_proto_workaround::protobuf::Timestamp;
+use ydb_grpc::ydb_proto::topic::{Consumer, SupportedCodecs};
+
+#[derive(serde::Serialize)]
+pub(crate) struct RawConsumer {
+    pub name: String,
+    pub important: bool,
+    pub read_from: Timestamp,
```

Comment:
Timestamp is protobuf type, it will problem on higher level if grpc layer will change in the future.

You can use std::time::SystemTime (prefer way) or Duration since UNIX_EPOCH

Reply:
IMO storing Duration is better. 
Just so we don't have to deal with conversion errors here.